app-shells/nushell/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=nushell
+PKGSEC=shells
+PKGDES="A modern and cross platform shell which provides strong structure processing ability and uses charts to handle data"
+PKGDEP="openssl gcc-runtime glibc"
+BUILDDEP="pkg-config openssl devel-base rustc llvm"
```

Comment:
Remove `openssl`, `devel-base` and `pkg-config`

Reply:
- `openssl` should be moved to PKGDEP, builddeps will be installed along with PKGDEP during build time - since they also need to link to these libraries.
- `devel-base` can be removed since it is built into the build environment.
- `pkg-config` might be used to retrieve the library paths, so you may not remove it.